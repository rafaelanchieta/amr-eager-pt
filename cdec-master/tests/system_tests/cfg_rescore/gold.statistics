-lm_nodes 8
-lm_edges 10
-lm_paths 4
