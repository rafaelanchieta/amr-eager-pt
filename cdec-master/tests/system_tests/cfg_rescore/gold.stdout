0 ||| the broccoli was eaten by John ||| Passive=1 Definite=1 ||| 2
0 ||| John ate the broccoli ||| Active=1 Definite=1 ||| 1.1
0 ||| broccoli was eaten by John ||| Passive=1 ||| 1
0 ||| John ate broccoli ||| Active=1 ||| 0.1
